infra/cifuzz/run_fuzzers.py

```diff
@@ -159,6 +164,28 @@ def run_fuzz_targets(self):
     return bug_found
 
 
+class CoverageTargetRunner(BaseFuzzTargetRunner):
+  """Runner that runs the 'coverage' command."""
+
+  @property
+  def quit_on_bug_found(self):
+    return False
+
+  def get_fuzz_targets(self):
+    """Returns fuzz targets in out_dir."""
+    # We only want fuzz targets from the root because during the coverage build,
+    # a lot of the image's filesystem is copied into /out for the purpose of
+    # generating coverage reports.
+    # TOOD(metzman): Figure out if we should do root_only for non-coverage runs.
```

Comment:
Maybe I should just go ahead and do this now? I don't think searching for fuzz targets recursively has real benefits at this time.